django/forms/widgets.py

```diff
@@ -696,27 +696,35 @@ class NullBooleanSelect(Select):
     """
     def __init__(self, attrs=None):
         choices = (
-            ('1', _('Unknown')),
-            ('2', _('Yes')),
-            ('3', _('No')),
+            ('unknown', _('Unknown')),
+            ('true', _('Yes')),
+            ('false', _('No')),
         )
         super().__init__(attrs, choices)
 
     def format_value(self, value):
         try:
-            return {True: '2', False: '3', '2': '2', '3': '3'}[value]
+            return {
+                True: 'true', False: 'false',
+                'true': 'true', 'false': 'false',
+                # For backward compatibility with previous versions
+                '2': 'true', '3': 'false',
+            }[value]
         except KeyError:
-            return '1'
+            return 'unknown'
 
     def value_from_datadict(self, data, files, name):
         value = data.get(name)
         return {
-            '2': True,
             True: True,
             'True': True,
-            '3': False,
             'False': False,
             False: False,
+            'true': True,
```

Comment:
Please check test coverage carefully. I don't see any tests failing with this line and the one below it removed. I didn't check the rest of the changes.